Admin/AdminInterface.php

```diff
@@ -201,11 +185,9 @@ public function getModelManager();
     public function getManagerType();
 
     /**
-     * @param string $context NEXT_MAJOR: remove this argument
-     *
      * @return ProxyQueryInterface
      */
-    public function createQuery($context = 'list');
+    public function createQuery();
```

Comment:
I think similar methods in `AdminExtension` and `AdminExtensionInterface` have also a `$context` argument, which should also be removed.